test-three/side-menu-app/src/components/CollapsableMenuItem.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import Collapse from '@material-ui/core/Collapse';
+import ExpandLess from '@material-ui/icons/ExpandLess';
+import ExpandMore from '@material-ui/icons/ExpandMore';
+//icons (material ui)
+import SettingsOutlinedIcon from '@material-ui/icons/SettingsOutlined';
+import AssessmentOutlinedIcon from '@material-ui/icons/AssessmentOutlined';
+import BrushOutlinedIcon from '@material-ui/icons/BrushOutlined';
+import TocOutlinedIcon from '@material-ui/icons/TocOutlined';
+import FiberManualRecordIcon from '@material-ui/icons/FiberManualRecord';
+
+import '../styles/style.css';
+
+
+
+export default function CollapsableMenuItem({ name, subItems }) {
+
+  // state to collapse/open the submenu
+  const [open, setOpen] = React.useState(true);
+  const handleClick = () => {
+    setOpen(!open);
+  };
+
+  // select sub-menu items
+  const [selectedIndex, setSelectedIndex] = React.useState(null)
+  const updateSelected = (index) => {
+    setSelectedIndex(index);
+  }
+
+
+  return (
+    <>
+      <ListItem button className="subOptionsButton" onClick={handleClick}>
+        <ListItemIcon>
+          {name === "Account" && <SettingsOutlinedIcon className="leftIcon" />}
```

Comment:
Same comment as before here. Do you think we can possibly hard code this somewhere else so that the component is cleaner?

Reply:
I have fixed this by using a separate component to handle the icons. (same as earlier comment)